students/Ben_freeman/lesson09/pngdiscover.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+
+big_list = []
+
+
+def dive(directory):
+    global big_list
+    contents = os.listdir(directory)
+    picture_holder = []
+    for items in contents:
+        #print(items)
+        try:
+            dive(items)
+        except:
+            if ".py" in items:
+                picture_holder.append(items)
+    print(picture_holder)
+    big_list.append(picture_holder)
```

Comment:
If you use _append()_ the entire _picture_holder_ list will be added as a **single element** into big list. You probably want to use _extend()_ that will add each element in _picture_holder_ as a separate element into _big_list_